### ISSUE TITLE
 Consolidate/merge/sort log lines from multiple files

### ISSUE THREAD

**Issue by natenho** [zarunbal/LogExpert#41]
# Feature

Log files can come from multiple servers for the same application. The merge feature consolidates these logs in one with chronologicaly sorted lines. It can be a enhancement for "Multifile" mode.

**Comment by zarunbal** [comment]
Thanks for your suggestion, do you mean this feature for static log files?

**Comment by natenho** [comment]
Yes, for static log files would be a good enhancement. It can be tricky to merge multiple files on the fly, if I got what you've meant.

**Comment by dm413** [comment]
I second this -- sorting would be very useful. But it's not limited to multiple files, and it's not limited to timestamp columns. It would be nice to be able to sort on any column. I envision the usual sorted grid behavior -- click on the column header and it sorts by that column. Click again and it sorts the other direction. 

I agree static log files is sufficient.

**Comment by steynru** [comment]
Are Sorting for static files under consideration?

**Comment by d4e666** [comment]
I would like this feature too since we now have a single log split on file size, merging them would mean I don't have to look at the filestamps to find the correct one (certainliy when an even spans multiple of these log files).

it would be nice if ther is a config stating how thse files are recognized (e.g. the main file has extension .log while all consequal files are .log.1 .log.2 and so forth.

**Comment by Hirogen** [comment]
> I would like this feature too since we now have a single log split on file size, merging them would mean I don't have to look at the filestamps to find the correct one (certainliy when an even spans multiple of these log files).
> 
> it would be nice if ther is a config stating how thse files are recognized (e.g. the main file has extension .log while all consequal files are .log.1 .log.2 and so forth.

the multifile function, already supports this, if you have a regex for the files and how they are saved on disk, the multifile function loads them one after the other
https://github.com/LogExperts/LogExpert/wiki/Settings#multi-file
https://github.com/LogExperts/LogExpert/wiki/MultiFile
